src/tokens/ERC1155/presets/lootbox/IERC1155Lootbox.sol

```diff
@@ -76,7 +76,7 @@ interface IERC1155LootboxSignals {
     error AllBoxesOpened();
 
     /// @notice Emits when a user make a commitment
-    event Commit(address user, uint256 blockNumber);
+    event Commit(address indexed user, uint256 blockNumber);
```

Comment:
Let's not index events. It costs more gas and doesn't give any benefit on our indexer since we filter on data.